services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -70,13 +72,28 @@ def _get_size_of_value(value: tuple[ItemConcreteValue | None, SetKWargs | None])
 )
 
 
+class OutputCallbacksWrapper(OutputsCallbacks):
```

Comment:
as I said... this thing should live entirely in the dynamic sidecar. nobody else needs it.